message_app/templates/messages.html

```diff
@@ -42,10 +37,11 @@
         <!-- Socket IO JS -->
         <script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/4.0.1/socket.io.js" integrity="sha512-q/dWJ3kcmjBLU4Qc47E4A9kTB4m3wuTY7vkFJDTZKjTs8jhyGQnaUrxa0Ytd0ssMZhbNua9hE+E7Qv1j+DyZwA==" crossorigin="anonymous"></script>
```

Comment:
I asked you to store this library on the server right?

Reply:
Yes. I forgot to store it. Just did in the last commit.